pkg/build/controller/strategy/docker.go

```diff
@@ -39,6 +39,7 @@ func (bs *DockerBuildStrategy) CreateBuildPod(build *buildv1.Build, additionalCA
 
 	privileged := true
 	strategy := build.Spec.Strategy.DockerStrategy
+	typeFile := v1.HostPathType("File")
```

Comment:
Shouldn't this be a constant in the API?

Reply:
It turns out we do have a constant on `corev1`. I have sent over a commit to address this.